cont3xt/integration.js

```diff
@@ -597,16 +621,17 @@ class Integration {
       res,
       sent: 0,
       total: 0, // runIntegrationsList will fix
-      resultCount: 0 // sum of _cont3xt.count from results
+      resultCount: 0, // sum of _cont3xt.count from results
+      queriedSet: new Set()
     };
     res.write('[\n');
-    res.write(JSON.stringify({ success: true, itype, query, sent: shared.sent, total: integrations.length, text: 'more to follow' }));
+    res.write(JSON.stringify({ purpose: 'init', success: true, indicator: { itype, query }, sent: shared.sent, total: integrations.length, text: 'more to follow' }));
     res.write(',\n');
 
     const issuedAt = Date.now();
     // end data write and create audit log when finished
     const finishWrite = () => {
-      res.write(JSON.stringify({ finished: true, resultCount: shared.resultCount }));
+      res.write(JSON.stringify({ purpose: 'finish', finished: true, resultCount: shared.resultCount }));
```

Comment:
can finished: true be removed because of purpose?

Reply:
Yep, I'll do that now